conf/application.conf

```diff
@@ -40,7 +40,7 @@ akka {
 # ~~~~~
 # The secret key is used to sign Play's session cookie.
 # This must be changed for production, but we don't recommend you change it in this file.
-play.http.secret.key = "changeme"
+play.http.secret.key = "tZhx=zlpcMsSX<WNCm=u4<q`S:I^YwAsScO7]FMr0:jEwjLsq73xesM@5a7p1SFo"
```

Comment:
Do **NOT** put production secrets into the repository...

Reply:
I have forgotten to change it back. It has been removed now. Thanks!